images/doppler-kubernetes-operator/metadata.hcl

```diff
@@ -0,0 +1,9 @@
+name                = "doppler-kubernetes-operator"
```

Comment:
This file isn't used anymore and can be deleted - everything should be contained in the `metadata.yaml`.

If monopod is still generating it I think a quick `cd monopod && go install` will fix that up.